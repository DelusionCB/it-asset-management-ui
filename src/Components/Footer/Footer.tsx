import React from 'react';
import './index.scss'
import logo from '../../Assets/Images/varha_logo_white.webp';

function Footer (): JSX.Element {
    return (
        <footer className='footer'>
            <img alt='Varhan logo' height={80} src={logo} />
        </footer>
    );
}

export default Footer;
