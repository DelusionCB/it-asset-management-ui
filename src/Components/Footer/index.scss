.footer {
    text-align: center;
    background-color: var(--primary-color);
    color: var(--white-color);
    padding: 2rem;
    margin-top: 1rem;
    position: relative;
    bottom: 0;
    left: 0;
    width: 100%;
    font-size: 1.25rem;
    a {
        color: var(--white-color);
    }
    .-image {
        padding: 1rem;
    }
    .-info {
        p {
            margin: 0;
            align-self: center;
        }
        display: flex;
        justify-content: space-evenly;
        @media screen and (max-width: 900px) {
            flex-direction: column;
        }
    }
}
