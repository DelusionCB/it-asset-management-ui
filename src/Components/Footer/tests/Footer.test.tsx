import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import Footer from '../Footer';
import '@testing-library/jest-dom'

describe('<Footer />', () => {
    let renderer: TestRenderer.ReactTestRenderer;

    // Render the component before tests
    beforeAll(() => {
        renderer = TestRenderer.create(
            <Footer />,
        );
    });
    describe('Snapshot', () => {
        it('renders correctly', () => {
            expect(renderer.toJSON()).toMatchSnapshot();
        })
    })
})
