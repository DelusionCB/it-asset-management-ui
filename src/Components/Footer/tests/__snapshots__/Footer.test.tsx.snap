// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Snapshot renders correctly 1`] = `
<footer
  className="footer"
>
  <img
    alt="Varhan logo"
    height={80}
    src=""
  />
</footer>
`;
